File: lib/src/main/java/uk/iatom/rfm69/registers/values/RxTimeout1.java
package uk.iatom.rfm69.registers.values; import uk.iatom.rfm69.registers.IRegisterValue;

public enum RxTimeout1 implements IRegisterValue {
    

    // RegRxTimeout1),
    RXSTART_VALUE(0x00); // Default),;

    private byte val;
    
    RxTimeout1(int val) {
        this.val = (byte) val;
    }

    @Override public byte val() {
        return val;
    }
}

File: lib/src/main/java/uk/iatom/rfm69/registers/values/RxTimeout1.kt
package uk.iatom.rfm69.registers.values

import uk.iatom.rfm69.registers.IRegisterValue

enum class RxTimeout1(`val`: Int): IRegisterValue { // RegRxTimeout1),
RXSTART_VALUE(0x00);


    // Default),;
    private val `val`: Byte

    init {
        this.`val` = `val`.toByte()
    }

    override fun `val`(): Byte {
        return `val`
    }
}
